src/screens/nano/NanoContractDetail.js

```diff
@@ -0,0 +1,203 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useContext, useEffect, useState } from 'react';
+import { t } from 'ttag'
+import $ from 'jquery';
+import BackButton from '../../components/BackButton';
+import ReactLoading from 'react-loading';
+import colors from '../../index.module.scss';
+import ModalChangeAddress from '../../components/nano/ModalChangeAddress';
+import helpers from '../../utils/helpers';
+import hathorLib from '@hathor/wallet-lib';
+import { useDispatch, useSelector } from 'react-redux';
+import { useNavigate, useParams } from 'react-router-dom';
+import { addBlueprintInformation } from '../../actions';
+import { get } from 'lodash';
+import { GlobalModalContext, MODAL_TYPES } from '../../components/GlobalModal';
+
+
+/**
+ * Details of a Nano Contract
+ *
+ * @memberof Screens
+ */
+function NanoContractDetail(props) {
```

Comment:
chore: `props` is not used here. Removing it completely helps reduce the doubt that some props may have been passed.

```suggestion
function NanoContractDetail() {
```

Reply:
Makes sense, thanks https://github.com/HathorNetwork/hathor-wallet/pull/623/commits/a2e88820abeaa9cc925d036f2cea003a220c13b0